modtools/create-item.lua

```diff
@@ -1,5 +1,6 @@
 -- creates an item of a given type and material
 --author expwnent
+--In the year 2024 dikbutdagrate assumed the identity of tjudge1 and edited this file. Now it spawns vermin, pets, eggs, fish, raw fish, and remains correctly.
```

Comment:
cute, but this is what the git history is for. if we added a line like this for every edit, we'd have more header comments than code.